src/cli.py

```diff
@@ -0,0 +1,25 @@
+import yaml
```

Comment:
Please get familiar with import groups:

Imports should be grouped in the separate groups following order:
standard library imports
related third party imports
local application/library specific imports
You should put a blank line between each group of imports.

Here it should be:
import yaml 

import click

from interface.app import create_gui